addons/bestja_account_deletion/wizards.py

```diff
@@ -23,7 +23,16 @@ def delete_account(self):
 class ReportWizard(models.TransientModel):
     _name = 'bestja.report_account_deletion_wizard'
 
-    reason = fields.Text(string="Dlaczego chcesz usunąć konto?", required=True)
+    REASONS = [
```

Comment:
I remember Ania saying user should be able to choose multiple reason. I think your solution is very reasonable, but can you confirm not allowing multiple choices is acceptable?